website/content/en/preview/getting-started/getting-started-with-karpenter/karpenter-controllers.json

```
@@ -351,7 +351,7 @@
         "datasource": {
           "uid": "${datasource}"
         },
-        "definition": "label_values(controller_runtime_reconcile_errors_total, controller)",
+        "definition": "label_values(controller_runtime_reconcile_errors_total{job=~\"karpenter\"}, controller)",
```

```suggestion
        "definition": "label_values(controller_runtime_reconcile_errors_total{controller="$controller"}, controller)",
```

This matches what we have for the other `controller_runtime_...` metrics in this file